### ISSUE TITLE
Adding Monetary Value & Likeliness in % to Opportunity 

### ISSUE THREAD

**Issue by dulhaver** [frappe/erpnext#6588]
from the manual
https://frappe.github.io/erpnext/user/manual/en/CRM/opportunity

> Best Practice: Leads and Opportunities are often referred as your “Sales Pipeline” this is what you need to track if you want to be able to predict how much business you are going to get in the future. Its always a good idea to be able to track what is coming in order to adjust your resources.

I totally agree to that remark but the Opportunity in ERPNext does not live up to that in my eyes. 
You can not put a monetary value to it neither put a likeliness in % of the opportunity to becoming actual business.

I am aware that it's possible to open a Quotation from an opportunity but from my point of view this often is not possible yet, because you simply might not have enough detail (items, quantities) in that stage.

An example. A new potential customer turns up and says: "_I want to work with you and would like to place monthly orders of ~20.000 USD for garments at your factory starting from next March_".
So this is an opportunity (20.000 USD/mnth starting in march) and in order to understand your "Sales Pipeline" you'd need to have this value somewhere combined with a likeliness of that business coming in. Practically you do not know the item details yet though, so a Quotation is not an option yet.

**Comment by Devens** [comment]
I agree with Subject point requirement. We have added that as  custom fields. But, if we can have the same from out of the box, it can be more helpful.

**Comment by Pandiyan5273** [comment]
@dulhaver 

* Addressed through the Opportunity and Sales Pipeline enhancements in #15524.
* Added an independent, currency-aware **Opportunity Amount** field.
* Added a **Probability (%)** field to Opportunity.
* Opportunity Amount and Probability can be entered without adding item details or creating a Quotation.
* Opportunity Amount is used in **Sales Pipeline Analytics** and **Opportunity Summary by Sales Stage** reports.
